composer.json

```diff
@@ -253,7 +253,8 @@
         "phlak/semver": "^2.0",
         "symfony/dom-crawler": "^3.4 || ^4",
         "typhonius/acquia-php-sdk-v2": "~1.1",
-        "vlucas/phpdotenv": "^2.4"
+        "vlucas/phpdotenv": "^2.4",
+        "wamania/php-stemmer": "^3.0"
```

Comment:
`drupal/porter_stemmer` is hardcoded to require search module, and this package has many hundreds of thousands of installs.